agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1467,6 +1457,16 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<SSlaveSocke
         // Add the slave socket to the backhaul configuration
         m_sConfig.slave_iface_socket[soc->sta_iface] = soc;
 
+        if (!m_agent_ucc_listener && request->certification_mode() && m_sConfig.ucc_listener_port != 0) {
+            m_agent_ucc_listener = std::make_unique<agent_ucc_listener>(
+                *this, m_sConfig.ucc_listener_port, m_sConfig.vendor, m_sConfig.model,
+                m_sConfig.bridge_iface, cert_cmdu_tx);
+            if (m_agent_ucc_listener && !m_agent_ucc_listener->start("ucc_listener")) {
+                LOG(ERROR) << "failed start agent_ucc_listener";
+                return false;
+            }
+        }
```

Comment:
What problem is solved by starting the UCC listener from `handle_slave_backhaul_message` instead of `init`?

Reply:
That on the init function we don't have certification mode enable flag from UCI.
So if we run UCC Listener we interfere with other applications on the platform that are using that port.